src/open_inwoner/utils/ckeditor.py

```diff
@@ -52,6 +53,9 @@ def get_product_rendered_content(product):
 
             element.attrs["class"] = class_name
 
+            if tag == "h2":
+                element.attrs["id"] = slugify(element.text)
```

Comment:
I think we should namespace the ID's this generates so we don't generate collisions with whatever ID's we might be using ourselves. Like with a prefix:

```py
element.attrs["id"] = f"subheader-{slugify(element.text)}"
```

(also theoretically this could generate duplicates if titles would be repeated, but let's not worry about that)